### ISSUE TITLE
A11y_Azure Data Studio_Azure Menu _Screenreader: Screenreader does not announcing "Presse Shift+F10 for expand Azure Menu" when keyboard focus is on Azure Tree Item.

### ISSUE THREAD

**Issue by harshgangrade111** [microsoft/azuredatastudio#14171]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudio_Feb2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG3.3.2;#Screenreader;#JAWS;#NVDA;#STP;#AINotInScope;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.26.0-insider (user setup)
Windows Version: Windows10
Screenreader: JAWS, NVDA

**Repro Steps:**
1.Launch Azure Data Studio insiders application.
2.Tab to Connections from left menu and in servers button and hit enter on "New Connection" button.
3.Connections section will get open.
4.Tab to "Browse Preview" Tab item and hit enter.
5.Tab to "Azure" tree item.
6.Start JAWS and check jaws is announcing shortcut key Shift+F10 for expand Azure Menu or not and verify the issue..

**Actual:**
Screenreader does not announcing "Presse Shift+F10 for expand Azure Menu" when keyboard focus is on Azure Tree Item.

**Expected:**
When keyboard focus is on Azure Tree Item screenreader Should also announce shortcut key detail that "Press Shift+F10 for expand Azure Menu".

**User Impact:**
Screenreader users will not get information regarding how to expand Azure Menu if screenreader does not announce press Shift+F10 for expand Azure menu.

**Comment by alanrenmsft** [comment]
@harshgangrade111 I don't think this is a reasonable expectation, Please provide the link to the requirement and examples implementing it and then reactivate the issue.

Thanks

**Comment by Roopesh-T** [comment]
Closing the bugs after discussion with PWD team, as the items given as menu items the functionality is know without announcement.

**GitHub Tags**:#Closed; #a11ysev4;#A11yusable;
